Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -99,11 +100,24 @@ public static void CreateMixedRealityToolkitGameObject()
         {
             Selection.activeObject = MixedRealityToolkit.Instance;
             Debug.Assert(MixedRealityToolkit.IsInitialized);
+            AssureCamera();
             var playspace = MixedRealityToolkit.Instance.MixedRealityPlayspace;
             Debug.Assert(playspace != null);
             EditorGUIUtility.PingObject(MixedRealityToolkit.Instance);
         }
 
+        private static void AssureCamera()
```

Comment:
This should be handled in [CameraCache](https://github.com/XRTK/XRTK-Core/pull/94/files?file-filters%5B%5D=.asset&file-filters%5B%5D=No+extension#diff-17eef5d7dd71af91d052c7e5097ab064R23)